Bio/SeqFeature.py

```diff
@@ -253,7 +261,7 @@ def __repr__(self):
         return answer
 
     def __str__(self):
-        """A readable summary of the feature intended to be printed to screen."""
+        """Return the full feature's sequence as a python string."""
```

Comment:
I wasn't imperative, what about `Return a summary of the feature intended to be printed to screen.`?